Content.Server/Dragon/DragonRiftSystem.cs

```diff
@@ -90,7 +90,16 @@ public override void Update(float frameTime)
             if (comp.SpawnAccumulator > comp.SpawnCooldown)
             {
                 comp.SpawnAccumulator -= comp.SpawnCooldown;
-                var ent = Spawn(comp.SpawnPrototype, xform.Coordinates);
+                // Delta-v part begins
+                comp.SpawnEliteAccumulator += 1;
+                var entSpawnPrototype = comp.SpawnPrototype;
+                if (comp.SpawnEliteAccumulator >= comp.SpawnEliteFrequency)
```

Comment:
Continuation of [this suggestion](https://github.com/DeltaV-Station/Delta-v/pull/4826/files#r2581861624).


```suggestion
                if (comp.SpawnElites && comp.SpawnEliteAccumulator >= comp.SpawnEliteFrequency)
```